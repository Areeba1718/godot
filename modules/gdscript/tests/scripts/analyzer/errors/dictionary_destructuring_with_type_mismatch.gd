func test():
    var a: int
    var b: int
    var { foo = a, bar = b } = { foo = "hello", bar = 42 }
