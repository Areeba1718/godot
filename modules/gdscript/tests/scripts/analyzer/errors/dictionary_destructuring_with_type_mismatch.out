GDTEST_ANALYZER_ERROR
Mismatched type "int" with key "foo" in the destructuring expression, expected "String".
